.rhcicd/frontend.yml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: user-preferences-frontend
+objects:
+  - apiVersion: cloud.redhat.com/v1alpha1
+    kind: Frontend
+    metadata:
+      name: user-preferences
+    spec:
+      envName: ${ENV_NAME}
+      title: User preferences
+      deploymentRepo: https://github.com/RedHatInsights/user-preferences-frontend
+      API:
+        versions:
+          - v1
+      frontend:
+        paths:
+          - /apps/user-preferences
+      image: ${IMAGE}:${IMAGE_TAG}
+      navItems:
+        - title: "Email preferences"
+          expandable: true
+          routes:
+            - appId: "userPreferences"
+              title: "Application Services"
+              href: "/settings/user-preferences/notifications/application-services"
+            - appId: "userPreferences"
+              title: "Openshift"
+              href: "/settings/user-preferences/notifications/openshift"
+            - appId: "userPreferences"
+              title: "Red Hat Enterprise Linux"
+              href: "/settings/user-preferences/notifications/rhel"
+            - appId: "userPreferences"
+              title: "Edge"
+              href: "/settings/user-preferences/notifications/edge"
+            - appId: "userPreferences"
+              title: "Console"
+              href: "/settings/user-preferences/notifications/console"
+      module:
+        manifestLocation: "/apps/user-preferences/fed-mods.json"
+        modules:
+          - id: "user-preferences-email"
+            module: "./RootApp"
+            routes:
+              - pathname: /user-preferences/notifications
```

Comment:
I think this should just be `/user-preferences` and `/user-preferences/email` according to [fed-mods](https://github.com/RedHatInsights/cloud-services-config/blob/0c9ee09eca4c7d6791cbd8054b74c3e914e9f334/chrome/fed-modules.json#L25)